application/templates/cms/create_dimension.html

```diff
@@ -37,12 +37,11 @@ <h1 class="heading-medium">Create new dimension</h1>
                     {{ render_textarea_field(form.summary, rows='7', cols='100', disabled=form_disabled) }}
 
                 {% endblock fields %}
-                <ul class="button-group" data-topbar data-options="sticky_on: large">
-                    <li>
-                        <button type="submit" class="button" name="save">Save
-                        </button>
-                    </li>
-                </ul>
+
+                <button type="submit" class="button" name="save">
```

Comment:
@TheDoubleK we could remove `type="submit"` here, as that’s the default. Also probably doesn’t need a `name` attribute, unless that's used by any javascript?